libs/auth/scripts/src/generate_machine_label_qr_code_contents.ts

```diff
@@ -0,0 +1,124 @@
+import { Readable } from 'node:stream';
+import yargs from 'yargs/yargs';
+import { extractErrorMessage } from '@votingworks/basics';
+
+import { signMessage } from '../../src/cryptography';
+import { FileKey } from '../../src/keys';
+import { constructPrefixedMessage } from '../../src/signatures';
+
+interface CommandLineArgs {
+  privateKeyPath: string;
+  privateKeyPassphraseFilePath?: string;
+  serialNumber: string;
+}
+
+async function parseCommandLineArgs(
+  args: readonly string[]
+): Promise<CommandLineArgs> {
+  const argParser = yargs()
+    .options({
+      'private-key-path': {
+        description:
+          'The path to the VotingWorks label QR codes private key. ' +
+          'This is not the same key as our root private key.',
+        type: 'string',
+      },
+      'private-key-passphrase-file-path': {
```

Comment:
do we need this? I don't love encouraging the passphrase to be in a file on disk.

Reply:
Ah yeah I was thinking of the batch generation use case where you'd need to generate URLs for say a hundred machine IDs and thereby need to enter in the passphrase a hundred times. But I'm okay with that.

Reply:
https://github.com/votingworks/vxsuite/pull/5768/commits/c6a00dd55f08fd1b98dddcacca1f01dee575eff3